apps/api/package.json

```diff
@@ -6,6 +6,8 @@
   "description": "Trading Simulator Server for hosting trading competitions",
   "main": "dist/src/index.js",
   "scripts": {
+    "format": "prettier --write . --ignore-path=../../.prettierignore",
```

Comment:
the way to handle prettier and eslint is a bit of a debate. i used [vercel's turbo monorepo](https://github.com/vercel/turborepo/blob/main/packages/turbo-workspaces/package.json#L25) as a reference:
- they install eslint and prettier at a global level
- each package defines the `lint` script
- each pacakge defines the `format` script (and passes the root prettier ignore file, unless overridden within the package's own prettier ignore file)
- the `turbo.json` and root `package.json` lets you run each subpackage's `lint` or `format` commands